great_expectations/cli/suite.py

```diff
@@ -87,7 +87,17 @@ def suite_edit(suite, datasource, directory, jupyter, batch_kwargs):
     )
 
 
-def _suite_edit(suite, datasource, directory, jupyter, batch_kwargs, usage_event):
+def _suite_edit(
+    suite,
+    datasource,
+    directory,
+    jupyter,
+    batch_kwargs,
+    usage_event,
+    actually_send_usage_message=True,
+):
+    # actually_send_usage_message flag is for the situation where _suite_edit is called by _suite_new().
+    # when called by _suite_new(), the flag will be set to False, otherwise it will default to True
```

Comment:
This `actually_send_usage_message` flag is needed for a specific case (when _suite_new() calls _suite_edit()). There might be a better way to do it, but this is the quick fix